packages/core/src/components/forms/label.tsx

```diff
@@ -42,7 +42,7 @@ export class Label extends React.PureComponent<ILabelProps, {}> {
         return (
             <label {...htmlProps} className={rootClasses}>
                 {text}
-                <span className={classNames(Classes.TEXT_MUTED)}>{helperText}</span>
+                <span className={classNames(Classes.TEXT_MUTED)}>{helperText ? " " + helperText : ""}</span>
```

Comment:
As the helper text is conditional, would it not be best to keep the span out of the DOM completely if helperText doesn't exist? e.g.
```
{helperText && <span className={classNames(Classes.TEXT_MUTED)}>{" " + helperText}</span>}
```

Reply:
Thanks for pointing this out Will. It seems cleaner to me. I’ll look if this affects the CSS. I can re-submit a PR tonight.

Reply:
Ah, the changes have broken the ability to add a React node as the helpertext e.g.
```js
<Label
  text="Username"
  helperText={
      <span style={{ display: 'block', fontSize: '80%' }}>
          blah
       </span>
  }
>
  ...
</Label>
```

This now displays as : `Username [object Object]`

I suggest changing to:

```js
{helperText && <span className={classNames(Classes.TEXT_MUTED)}>{" "}{helperText}</span>}
```